static/scss/grade_record.scss

```diff
@@ -109,11 +109,17 @@
   .school-name {
     align-self: start;
     margin-bottom: .625rem;
-    font-size: 1.5rem;
+    font-size: 2.5rem;
     line-height: 1.5;
 
     img {
-      width: 180px;
+      height: 40px;
+    }
+
+    .pipe {
+      vertical-align: sub;
+      font-size: 20px;
+      font-weight: 600;
```

Comment:
might want to rename this class -- although `.on` seems weird... is `mitxonedx` too long? 

Can we make this black? In your screenshot it's looking kind of gray. Otherwise 👍